app/controllers/repos_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ReposController < ApplicationController
+  def index
+    github_service = GithubService.new(current_user.token)
+    @repos = github_service.find_repos
```

Comment:
@jmejia i am not quite able to see the need for a PORO for Repo - but I have a suspicion I need one - not quite sure how to start that flow with my testing. Am I just sort of skipping over that part and getting the response right from the github service?